src/quacc/utils/calc.py

```diff
@@ -161,6 +161,18 @@ def run_ase_opt(
     if relax_cell and atoms.pbc.any():
         atoms = ExpCellFilter(atoms)
 
+    if optimizer_kwargs["use_TRICs"] and optimizer.__name__ != "Sella":
```

Comment:
Note: You mean `optimizer_kwargs.get("use_TRICs")` so that it works when that key isn't present at all.

Reply:
Yup, thanks